documentation/create_overview_doc/README.md

```diff
@@ -51,21 +51,27 @@ sh
 kt
 ```
 
+For flow map generation with Code2Flow, the tool currently supports the following languages, each with specific dependencies:
+
+- JavaScript: Requires Acorn for parsing. Ensure Acorn is installed and accessible in your environment.
+- Ruby: Uses Parser for parsing. Ensure the Parser gem is installed and accessible.
+- PHP: Employs PHP-Parser for parsing. PHP-Parser should be installed and accessible in your environment.
+- Python: No additional dependencies are required for Python projects.
+
+If your project uses a language not supported by Code2Flow for flow map generation, the documentation generation capabilities will still be available, but without the flow map visualization for those specific languages.
+
 ## Prerequisites
 
 Ensure the following tools are installed:
 
 - `bito` : https://github.com/gitbito/CLI
 
-- `mermaidcli` : https://github.com/mermaid-js/mermaid-cli
+- `Code2Flow:` : https://github.com/scottrogowski/code2flow
```

Comment:
**Issue**: The prerequisites section was updated to include Code2Flow but did not specify the version requirements or installation steps for Graphviz, which is necessary for Code2Flow's operation. <br> **Fix**: Update the prerequisites section to include specific version requirements for Graphviz and provide a link to the installation guide. <br> **Code Suggestion**: 
 ```
 +  - 'Graphviz': Required for generating flowcharts with Code2Flow. Ensure you have Graphviz (version >= 2.40.1) installed. [Graphviz Installation Guide](https://graphviz.org/download/)
 ```